third_party/3/pkg_resources.pyi

```diff
@@ -16,78 +16,53 @@ _InstallerType = Callable[[Requirement], Optional[Distribution]]
 _EPDistType = Union[Distribution, Requirement, str]
 _MetadataType = Optional[IResourceProvider]
 _PkgReqType = Union[str, Requirement]
-_DistFinderType = Callable[[str, _Importer, bool],
-                           Generator[Distribution, None, None]]
+_DistFinderType = Callable[[str, _Importer, bool], Generator[Distribution, None, None]]
 _NSHandlerType = Callable[[_Importer, str, str, types.ModuleType], str]
 
 def declare_namespace(name: str) -> None: ...
 def fixup_namespace_packages(path_item: str) -> None: ...
 
-
 class WorkingSet:
     entries = ...  # type: List[str]
     def __init__(self, entries: Optional[Iterable[str]] = ...) -> None: ...
     def require(self, *requirements: _NestedStr) -> Sequence[Distribution]: ...
     def run_script(self, requires: str, script_name: str) -> None: ...
-    def iter_entry_points(self, group: str, name: Optional[str] = ...) \
-                          -> Generator[EntryPoint, None, None]: ...
+    def iter_entry_points(self, group: str, name: Optional[str] = ...) -> Generator[EntryPoint, None, None]: ...
     def add_entry(self, entry: str) -> None: ...
     def __contains__(self, dist: Distribution) -> bool: ...
     def __iter__(self) -> Generator[Distribution, None, None]: ...
     def find(self, req: Requirement) -> Optional[Distribution]: ...
-    def resolve(self, requirements: Sequence[Requirement],
-                env: Optional[Environment] = ...,
-                installer: Optional[_InstallerType] = ...) \
-                -> List[Distribution]: ...
-    def add(self, dist: Distribution, entry: Optional[str] = ...,
-            insert: bool = ..., replace: bool = ...) -> None: ...
+    def resolve(self, requirements: Sequence[Requirement], env: Optional[Environment] = ..., installer: Optional[_InstallerType] = ...) -> List[Distribution]: ...
+    def add(self, dist: Distribution, entry: Optional[str] = ..., insert: bool = ..., replace: bool = ...) -> None: ...
     def subscribe(self, callback: Callable[[Distribution], None]) -> None: ...
-    def find_plugins(self, plugin_env: Environment,
-                     full_env: Optional[Environment] = ...,
-                     fallback: bool = ...) \
-                     -> Tuple[List[Distribution],
-                              Dict[Distribution, Exception]]: ...
+    def find_plugins(self, plugin_env: Environment, full_env: Optional[Environment] = ..., fallback: bool = ...) -> Tuple[List[Distribution], Dict[Distribution, Exception]]: ...
```

Comment:
Actually, our style guidelines now say to limit lines to 130 characters. This one is 177.